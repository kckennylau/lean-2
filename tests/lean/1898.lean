def X (R : Type) [H : has_zero R] := H.0
