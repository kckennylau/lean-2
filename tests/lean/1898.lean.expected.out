1898.lean:1:38: error: invalid projection, index must be greater than 0
1898.lean:1:4: error: don't know how to synthesize placeholder
context:
R : Type,
H : has_zero R
‚ä¢ Sort ?
